commcare_connect/opportunity/api/serializers.py

```diff
@@ -178,6 +185,18 @@ def get_catchment_areas(self, obj):
         catchments = CatchmentArea.objects.filter(opportunity_access=opp_access)
         return CatchmentAreaSerializer(catchments, many=True).data
 
+    @lru_cache
+    def _get_flags(self, obj):
+        return OpportunityVerificationFlags.objects.filter(opportunity=obj).first()
```

Comment:
should this use `.get` instead or even `obj.opportunityverificationflags`? It's a `OneToOne` model, we can guarantee there is only one so `filter` and `first` seem unnecessary. Given that we can use normal accessor syntax, this whole cached get may be unnecessary, since it will be loaded the first time its accessed and already saved on the object the later times (rather than saved in redis)

Reply:
Good point, I totally overlooked it. I have updated the code and also addressed the rest of the comments too.